manifests/install/package.pp

```diff
@@ -81,6 +81,18 @@
         }
       }
 
+      'Suse'   : {
+        if $vmwaretools::redhat_install_devel == true {
```

Comment:
I'm unsure about this - perhaps the parameter should be renamed to something more generic? Maybe `install_devel`?